domains/aakhilv.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "A personal portfolio site for all my projects and commissions.",
+  "repo": "https://github.com/aakhilv/portfolio",
+  "owner": {
+    "username": "aakhilv",
+    "email": "bluninja165@gmail.com",
+    "twitter": "corruptblu"
+  },
+  "record": {
+    "CNAME": "aakhilv.github.io/portfolio"
```

CNAME has to be a valid hostname. Full urls wont work.

```suggestion
    "CNAME": "aakhilv.github.io"
```

Then after the domain has been published, in the `portfolio` repo settings, set the custom domain to `aakhilv.is-a.dev`